src/components/ChecToast.vue

```diff
@@ -16,16 +25,18 @@
 
 <script>
 import SvgCloseIcon from '../assets/svgs/close-icon.svg';
+import ChecIcon from './ChecIcon.vue';
 
 export default {
-  name: 'ChecNotification',
+  name: 'ChecToast',
   components: {
     SvgCloseIcon,
+    ChecIcon,
   },
   props: {
     /**
-     * The style variant for the notification. One of "success", "error", "warning", "info". Default is "info"
-     */
+         * The style variant for the notification. One of "success", "error", "warning", "info". Default is "info"
+         */
```

Comment:
Your IDE is doing something strange here I'm guessing? These comments were correct before.

Reply:
yeah weird fixed - locally